client/lib/abtest/active-tests.js

```diff
@@ -228,4 +228,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	secureYourBrand: {
+		datestamp: '20201008',
+		variations: {
+			test: 50,
+			control: 50,
+		},
+		defaultVariation: 'control',
+		allowExistingUsers: true,
```

Comment:
I think we can set this to false since this test is for `EN` only, and logged in users are going to be redirected to the new onboarding flow.